paddle/fluid/operators/jit/helper.cc

```diff
@@ -66,7 +66,8 @@ const char* to_string(KernelType kt) {
     ONE_CASE(kEmbSeqPool);
     ONE_CASE(kSgd);
     default:
-      PADDLE_THROW("Not support type: %d, or forget to add it.", kt);
+      PADDLE_THROW(platform::errors::Unimplemented(
+          "Not support type: %d, or forget to add it.", kt));
```

Comment:
type of what?   is this user `forget`? if user can not `add` here, I think we can only tell users `Upsupported`

Reply:
Done